lib/house.rb

```diff
@@ -0,0 +1,42 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @above_market_average = false
```

Comment:
this doesn't need to be an instance variable-- you can just have this be a method

```
def above_market_average
     @price > 500_000
end
```